src/order.py

```diff
@@ -1,169 +1,44 @@
+from eip712.messages import EIP712Message
 import time
-import eip712_structs
-from typing import Set, Optional
-
-import requests
-from eip712_structs import Address, Boolean, Bytes, String, Uint, EIP712Struct
-from eth_account.messages import encode_defunct
+from hexbytes import HexBytes
 from web3.auto import w3
-
-# TODO - all the global declarations here should become environment vars passed in
-#  as arguments at runtime. Specifically,
-#  `network`/`chain_id`, `private_key`, `base_url`, `stable_coins`, `verifyingContract`
-network = "xdai"
-if network == "mainnet":
-    base_url = "https://protocol-mainnet.gnosis.io/api/v1/"
-    private_key = "XXXX"
-    public_address = "0xE2A4dFC3Cab6dCdb198C83066B93110D62d95F09"
-    domain = eip712_structs.make_domain(
-        name='Gnosis Protocol',
-        version='v2',
-        chainId='1',
-        verifyingContract="0x9008D19f58AAbD9eD0D60971565AA8510560ab41"
-    )
-else:
-    base_url = "https://protocol-xdai.gnosis.io/api/v1/"
-    private_key = "XXXX"
-    public_address = "0x89145F922BB420453bCd8a7Be87dA830C0941f22"
-    domain = eip712_structs.make_domain(
-        name='Gnosis Protocol',
-        version='v2',
-        chainId='100',
-        verifyingContract="0x9008D19f58AAbD9eD0D60971565AA8510560ab41"
-    )
-
-ETH = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"
-
-WETH_ADDRESS = {
-    "mainnet": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
-    "xdai": "0xe91d153e0b41518a2ce8dd3d7944fa863463a97d"
-}
-
-WETH = WETH_ADDRESS[network]
-
-stable_coins = {
-    "0xddafbb505ad214d7b80b1f830fccc89b60fb7a83": 6,  # USDC
-    "0xe91d153e0b41518a2ce8dd3d7944fa863463a97d": 18,  # wxDAI
-    "0x4ecaba5870353805a9f068101a40e0f32ed605c6": 6  # USDT
-}
```

Comment:
Could we please keep these constants somewhere maybe in a file called `constants.py`. At some point soon, I would expect them to be reintroduced into the bot logic.

Reply:
Yes, for the bot I have another file where I keep these